spec/cli_spec.rb

```diff
@@ -1,25 +1,64 @@
 require_relative '../lib/cli.rb'
+require_relative '../lib/content.rb'
+require 'pry'
 
 describe 'call' do
   before do
+    allow($stdin).to receive(:gets)
     allow($stdout).to receive(:puts)
-
     @cli = Cli.new
   end
 
   it 'clears screen before printing menu' do
     expect(@cli).to receive(:clear_screen)
+    allow(@cli).to receive(:gets).and_return('nil')
```

Comment:
If this was really nil, you wouldn't need it to be returned and it wouldn't be in quotes

Reply:
@ssunday is it really returning an empty space and `allow(@cli).to receive(:gets).and_return(' ')` should fix this?

Reply:
If that's what it's supposed to be doing!

Reply:
commit 8e131fd9e6d8f8cc6525b407331850489f6d58bb